apis/infix-en_GB/atrium-api-infix-en_GB/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/SequenceSubjectChangerSamples.kt

```diff
@@ -0,0 +1,57 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class SequenceSubjectChangerSamples {
+
+    @Test
+    fun asIterableFeature() {
+        val sequence = sequenceOf(1, 2, 3)
+        expect(sequence) asIterable o toContain o inGiven order and only the values(1, 2, 3)
+        //                  |                       | order specifier
+        //                  |  subject is now type Iterable<Int>
```

Comment:
please use the same comments as in api-fluent. This way we can search & replace in the future if necessary. So:
```suggestion
        //                  |  subject is now of type Iterable<Int>
```